avm/res/event-grid/domain/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,78 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-eventgrid.domains-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'egdmin'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-nestedDependencies'
+  params: {
+    virtualNetworkName: 'dep-${namePrefix}-vnet-${serviceShort}'
+    managedIdentityName: 'dep-${namePrefix}-msi-${serviceShort}'
+    location: location
+  }
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+@batchSize(1)
+module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem' ]: {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-test-${serviceShort}-${iteration}'
+  params: {
+    name: '${namePrefix}${serviceShort}001'
+    location: location
+    inboundIpRules: []
+    privateEndpoints: [
+      {
+        privateDnsZoneResourceIds: [
+          nestedDependencies.outputs.privateDNSZoneResourceId
+        ]
+        service: 'domain'
+        subnetResourceId: nestedDependencies.outputs.subnetResourceId
+        tags: {
+          'hidden-title': 'This is visible in the resource name'
+          Environment: 'Non-Prod'
+          Role: 'DeploymentValidation'
+        }
+      }
+    ]    
+    managedIdentities: {
+      systemAssigned: true
+    }
```

Comment:
It appears all those parameters are optional and should hence not be part of the 'min' test (but instead max). I'd hence suggest to remove them and regenerate the readme.
```suggestion
```

Reply:
That was one of my doubts because I understand that "default" and "waf-aligned" tests should be compliant with all the PSRules. In this case if I avoid inserting this part, I receive exceptions in the PSRules

Reply:
Hey @fabmas, 
then I know the confusion you're referring to. What you're seeing is actually not PSRule complaining about those parameters (or rather their lack of), but outright failing, as there is a [know issue](https://github.com/microsoft/PSRule/issues/1640) in PSRule that is currently being addressed. It's the reason why you can see in many of our test files in AVM that it says something along the lines of 'Workaround for PSRule' where nullable parameters are introduced with a dummy null value to the test file.

E.g.

https://github.com/Azure/bicep-registry-modules/blob/3c5f62204b5ef47d2c396fe43588d355c4a712df/avm/res/batch/batch-account/tests/e2e/defaults/main.test.bicep#L48-L66